Scripts/script-ExtractEmail.yml

```diff
@@ -0,0 +1,51 @@
+commonfields:
+  id: ExtractEmail
+  version: -1
+name: ExtractEmail
+script: |
+  var text = args.text;
+  if (typeof text !== 'string') {
+      text = JSON.stringify(args.text).replace(/\\n/g,' ');
+  }
+  var matches = {};
+  var reg = /[a-z0-9]+[_a-z0-9\.-]*[a-z0-9]+@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})/gmi;
+
+  var whitelist = getCSVListAsArray('Indicators Whitelist');
+  var found;
+
+  while (found = reg.exec(text)) {
+   if (found && whitelist.indexOf(found[0]) < 0) {
+       matches[found[0]] = true;
+   }
+  }
+
+  var uniqueMatches = Object.keys(matches);
+  var emailContext = { Email: [] };
```

Comment:
According to our taxonomy, it should go under `Account.Email` and not `Email.Data`